Proton/Sniffs/Spacing/ArrowFunctionSpacingSniff.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Proton\Sniffs\Architecture;
```

Comment:
We should have another namespace for this sniff, maybe "Style"

Reply:
Yes, that's copy-paste, it should be aligned to the directory name `Spacing`